README.md

```diff
@@ -1,28 +1,43 @@
-# purescript-ace
+# Ace
 
-[![Latest release](http://img.shields.io/github/release/slamdata/purescript-ace.svg)](https://github.com/slamdata/purescript-ace/releases)
-[![Build status](https://travis-ci.org/slamdata/purescript-ace.svg?branch=master)](https://travis-ci.org/slamdata/purescript-ace)
-[![Dependency status](https://img.shields.io/librariesio/github/slamdata/purescript-ace.svg)](https://libraries.io/github/slamdata/purescript-ace)
+[![CI](https://github.com/purescript-contrib/purescript-ace/workflows/CI/badge.svg?branch=main)](https://github.com/purescript-contrib/purescript-ace/actions?query=workflow%3ACI+branch%3Amain)
+[![Release](http://img.shields.io/github/release/purescript-contrib/purescript-ace.svg)](https://github.com/purescript-contrib/purescript-ace/releases)
+[![Pursuit](http://pursuit.purescript.org/packages/purescript-ace/badge)](http://pursuit.purescript.org/packages/purescript-ace)
+[![Maintainer: milesfrain](https://img.shields.io/badge/maintainer-milesfrain-teal.svg)](http://github.com/milesfrain)
 
 PureScript bindings for the [Ace code editor](http://ace.c9.io).
 
-## API
+## Installation
 
-The API is a low-level, type-safe, straightforward binding to Ace:
+Install `ace` with [Spago](https://github.com/purescript/spago):
 
-* All effectful JavaScript functions (i.e. virtually *all* functions!) are wrapped in the `Eff` monad for the appropriate effect type (`random`, `dom`, etc.).
-* All JavaScript parameter or return values which may be nullable are wrapped / unwrapped using `Maybe`.
-* PureScript record types are used for JavaScript objects.
-* JavaScript's fake sum types are mapped to PureScript's real sum types, which necessitates encoding / decoding.
+```sh
+spago install ace
+```
 
-## Examples
+## Quick start
 
-The `examples` directory contains an `index.html`, which contains a variety of examples of how to embed the Ace editor and configure it using the PureScript API.
+The quick start hasn't been written yet (contributions are welcome!). The quick start covers a common, minimal use case for the library, whereas longer examples and tutorials are kept in the [docs directory](./docs.)
 
-## Tests
+## Documentation
 
-This project has no real tests. However, most bound functions are exercised in the examples, to ensure the bindings and necessary wrapping, unwrapping, encoding, and decoding has been performed correctly.
+`ace` documentation is stored in a few places:
 
-## Documentation
+1. Module documentation is [published on Pursuit](https://pursuit.purescript.org/packages/purescript-ace).
+2. Written documentation and [the changelog](./docs/CHANGELOG.md) are kept in [the docs directory](./docs).
+3. Usage examples can be found in [the test suite](./test).
```

Comment:
(this is where those sections from docs/README can be moved)

Reply:
We'd essentially adjust the third bullet to point to the examples instead of the test suite, as that doesn't have any usage examples.